### PR TITLE
added "New On Otakugram" Card Slider

### PR DESCRIPTION
**Pull request by RektyRowdyy**

### CODE REVIEW

File: Otakugram-UI/frontend/src/components/Home/CardSlider/CardSlider.jsx [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+import "slick-carousel/slick/slick-theme.css";
+import { FaChevronLeft, FaChevronRight } from 'react-icons/fa';
+import './CardSlider.css'
+
+// Function to create custom arrows
+function CustomArrows(props) {
+
+    const { className, style, onClick, arrowType } = props;
+
+    if (arrowType === 'next') {
```
Comment (7373Mehar) [inline Otakugram-UI/frontend/src/components/Home/CardSlider/CardSlider.jsx]: Make a constant file and fetch 'next' from that place.

File: Otakugram-UI/frontend/src/components/Home/CardSlider/CardSlider.jsx [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+        return (
+            <FaChevronRight
+                className={className}
+                style={{ ...style, color: "white", right: "1%", zIndex: 1, height: "30px" }}
+                onClick={onClick}
+            />
+        );
+    } 
+    else if (arrowType === 'prev') {
```
Comment (7373Mehar) [inline Otakugram-UI/frontend/src/components/Home/CardSlider/CardSlider.jsx]: Make a constant file and fetch 'prev' from that place.

File: Otakugram-UI/frontend/src/components/Home/CardSlider/CardSlider.jsx [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+            />
+        );
+    }
+}
+
+export default function CardSlider() {
+
+    //Static Anime Data
+    const data = [
+        {
```
Comment (7373Mehar) [inline Otakugram-UI/frontend/src/components/Home/CardSlider/CardSlider.jsx]: entire array from constant file.

**Review by 7373Mehar — CHANGES_REQUESTED** [review]

**Comment by RektyRowdyy** [comment]
@7373Mehar updated UI and transferred all the static data to constants.jsx

**Review by 7373Mehar — APPROVED** [review]

**Pull request merged**